src/drv_wmr/wmr.c

```diff
@@ -454,8 +477,9 @@ static ohmd_device* open_device(ohmd_driver* driver, ohmd_device_desc* desc)
 	return (ohmd_device*)priv;
 
 cleanup:
-	if(priv)
+	if (priv) {
 		free(priv);
+	}
```

Comment:
I don't like this very much, being used to kernel coding style. But if we are going to make this a rule I'm ok with it.

Reply:
Fair enough, that's deffo a meism :)